### ISSUE TITLE
Recent election

### ISSUE THREAD

**Issue by noahdasanaike** [DIRKMJK/kiesraad#3]
Would you consider updating these to process the 2022 election?

**Comment by DIRKMJK** [comment]
Could you specify the problem you ran into when trying to scrape or parse recent election data?

**Comment by noahdasanaike** [comment]
There is no output when running the code for the 2022 EML files (or, at least, I must be doing it wrong)—same setup worked fine for previous years.

**Comment by DIRKMJK** [comment]
Thanks for clarifying. I tried with this zip of EML files: https://data.overheid.nl/dataset/verkiezingsuitslagen-gemeenteraad-2022#panel-resources (see link provided there under the heading ‘Verkiezingsuitslagen Gemeenteraad 2022 (EML)’); this one worked for me, although I haven’t checked the results. I think that in the past I’ve also come across EML data for a specific election divided over three separate downloads; in that case I had to download all three of them and save them in the source directory to make it work.
